values.yaml

```diff
@@ -966,15 +997,13 @@ meshGateway:
   hostPort: null
 
   # Resource settings for mesh gateway pods.
-  # NOTE: The use of a YAML string is deprecated. Instead, set directly as a
-  #       YAML map.
```

Comment:
Hmm, similar to above, why are we removing the comment? We still support both.

Reply:
yes, rebase mistake